tests/test_modules_buildwrappers.py

```diff
@@ -21,3 +21,18 @@ def test_basic(self):
         assert jjb_ws_cleanup['disable-deferred-wipeout'] is True
         assert jjb_ws_cleanup['check-parameter'] == 'DO_WS_CLEANUP'
         assert jjb_ws_cleanup['external-deletion-command'] == 'shred -u %s'
+
+    def test_empty_value(self):
+        filename = os.path.join(fixtures_path, 'prebuildcleanup_empty_value.xml')
+        root = get_xml_root(filename=filename)
+        assert root is not None
+        parent = []
+        prebuildcleanup(root, parent)
+        assert len(parent) == 1
+        assert 'workspace-cleanup' in parent[0]
+        jjb_ws_cleanup = parent[0]['workspace-cleanup']
+        assert jjb_ws_cleanup['dirmatch'] is True
+        assert jjb_ws_cleanup['disable-deferred-wipeout'] is True
+        assert 'DO_WS_CLEANUP' not in jjb_ws_cleanup
```

Comment:
this is just a value, not a key, so this assert will always be True, even without the fix

Reply:
Yes, miswritten that. Thanks for noticing it.